### DISCUSSION TITLE
SceneView background

### DISCUSSION THREAD

**Discussion by Methew5** [sceneview/sceneform-android#249] [Q&A]
Is it possible to use an IBL with SceneView (HDR file) and remove the background picture from SceneView, that comes with HDR file?

**Comment by ThomasGorisse** [comment]
Sure. You can load your .hdr from resources, assets and even from an http url.
You can also, instead, use ktx to include preprocessed indirect light and skybox values.

https://github.com/SceneView/sceneform-android/blob/e08a2d8d619baf5cc54cd8ea8292ab54f63648cc/samples/environment-lights/src/main/java/com/google/ar/sceneform/samples/environmentlights/MainActivity.kt#L59-L63

**Reply by Methew5** [reply]
I already use an HDR file, but with reflections, I get background, which I don't need, can I get rid of it?

**Comment by ThomasGorisse** [comment]
Try this:
```kotlin
sceneView.renderer?.removeEntity(sceneView.cameraStream.cameraStreamRenderable)
sceneView.renderer?.setSkybox(environment?.skybox)
```

**Reply by ThomasGorisse** [reply]
Sorry about that but I forgot the non AR usage on the environment part.
I edited the answer.

**Reply by Methew5** [reply]
Since SceneView doesn't have a camera, there is no cameraStream property in SceneView too, hence the first line doesn't work, as you mentioned above.
Second line helped me a lot, after setting up the environment, I just remove the skybox:
```
sceneView.renderer?.setSkybox(null)
```
Thank you :)

**Reply by ThomasGorisse** [reply]
> Since SceneView doesn't have a camera, there is no cameraStream property in SceneView too, hence the first line doesn't work, as you mentioned above.

I thought you was in an AR context.
BTW, if you didn't already, you should try it (AR without ArCamera). It makes an awesome result where you move with your phone to move inside the hdr virtual environment.

> Second line helped me a lot, after setting up the environment, I just remove the skybox:

Cool. Can you share screenshots?

**Reply by ThomasGorisse** [reply]
> BTW, if you didn't already, you should try it (AR without ArCamera). It makes an awesome result where you move with your phone to move inside the hdr virtual environment.

Maybe we could find a name for it: AR + VR = ? (not Mixed Reality since it means an other thing)
